source/Halibut.Tests/Support/TestAttributes/LatestAndPreviousClientAndServiceVersionsNoWebSocketPerfectNetworkTestCases.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Halibut.Tests.Support.TestCases;
+
+namespace Halibut.Tests.Support.TestAttributes
+{
+    public class LatestAndPreviousClientAndServiceVersionsNoWebSocketPerfectNetworkTestCases : LatestAndPreviousClientAndServiceVersionsNoWebSocketsTestCases
```

Comment:
We can review how we do variations through attributes. These were added as a first pass to give the support needed in this pr.

Reply:
Yep I think as we get more examples of the tests we need this can and may all change.